src/workerd/server/BUILD.bazel

```diff
@@ -179,6 +181,16 @@ kj_test(
     ],
 )
 
+kj_test(
+    src = "bundle-fs-test.c++",
+    deps = [
+        "//src/workerd/io",
```

Comment:
I don't think you need IO

Reply:
bundle-fs depends on worker-fs which depends on io-context, which is part of io